src/stdlib_io_disp.fypp

```diff
@@ -0,0 +1,125 @@
+#:include "common.fypp"
+#:set RIL_KINDS_TYPES = REAL_KINDS_TYPES + INT_KINDS_TYPES + LOG_KINDS_TYPES
+
+submodule (stdlib_io) stdlib_io_disp
+
+    use, non_intrinsic :: stdlib_strings, only: format_string
```

Comment:
`non_intrinsic` is the default, is there a reason for the extra verbosity?

Reply:
This may be my own bad habit, sorry, **I will remove it， too**.